webdriver-spec.html

```diff
@@ -791,6 +791,11 @@
  has an associated list of <dfn>remote end steps</dfn>.
  This provides the sequence of actions that a <a>remote end</a> takes
  when it receives a particular <a>command</a>.
+ All <dfn data-lt="RES-arguments"><a>remote end steps</a>
+ accept the arguments</dfn> <var>url variables</var> and <var>parameters</var>,
```

Comment:
Provide reference for the typing of these two arguments. I’m guessing that the _URL variables_ should be a _URI Templates variable-list_, and that _parameters_ should be a _JSON Object_.